### PR TITLE
KEYCLOAK-7127 third and final set of changes from Getting Started review

### PR DESCRIPTION
**Pull request by matthewhelmke**
The named JIRA task is to review and edit the Getting Started guide. This is my third and final set of changes.

I am intentionally working backwards through the guide, section by section, to see if I can also catch missing information while standardizing our language, grammar, and overall voice.

### CODE REVIEW

**Comment by matthewhelmke** [comment]
Ready for review.

**Comment by Pepo48** [comment]
I've gone through all 3 PRs by @matthewhelmke - everything seems to be OK, no important information was lost.

**Comment by matthewhelmke** [comment]
Thank you, @Pepo48.

Ping @stianst. The other two PRs are #377 and #378.

**Review by Pepo48 — DISMISSED** [review]

File: getting_started/topics/first-boot/distribution-files-community.adoc [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
-to run the {project_name} server.
+NOTE: This file can be downloaded from https://www.keycloak.org/downloads.html[Keycloak downloads].
 
-To unpack these files run the `unzip` or `gunzip` and `tar` utilities.
+The *keycloak-{project_version}.[zip|tar.gz]* file is the server-only distribution. It contains only the scripts and binaries to run the {project_name} server.
+
+Place the file in a directory you choose and use either the `unzip` or `tar` utility to extract it.
+
+.Linux/Unix
+[source]
```
Comment (stianst) [inline getting_started/topics/first-boot/distribution-files-community.adoc]: Should be:
[source,bash,subs=+attributes]

Using bash highlights the code properly and subs=+attributes allows variables within source blocks

**Review by stianst — CHANGES_REQUESTED** [review]
[source] blocks should set language (bash) and all blocks with variables needs subs=+attributes

File: getting_started/topics/first-boot/distribution-files-community.adoc (hunk 1 above)
Comment (matthewhelmke) [reply getting_started/topics/first-boot/distribution-files-community.adoc]: Will push a new commit momentarily.

**Comment by matthewhelmke** [comment]
@stianst all instances of the [source] tag are fixed in these files.

**Comment by matthewhelmke** [comment]
Bump. @stianst once you have a chance to review the changes you requested, we can merge and close KEYCLOAK-7127.

**Review by stianst — DISMISSED** [review]

**Review by mposolda — APPROVED** [review]

**Pull request merged**